charts/oes/values.yaml

```diff
@@ -338,6 +356,25 @@ sapor:
           file: /opt/opsmx/client.p12
           password: changeit
 
+    ## Set the below field to true if datasource configurations from platform
+    datasources:
+      platform: true
+
+    ## Set the below field to true if agent for kubernetes
+    kubernetes:
+      agent:
+        enabled: true
+        serverHostName: opsmx-controller-controller1
+        serverPort: 9001
+        caCertfile: /opt/opsmx/controller/ca.crt
+        certFile: /opt/opsmx/controller/cert/tls.crt
+        keyFile: /opt/opsmx/controller/cert/tls.key
+        image: quay.io/opsmxpublic/forwarder-agent:v20210118T161303
+      template:
+        path: /opt/opsmx/controller
+        kubectlTemplateFileName: kubeconfig.template
+        manifestTemplateFileName: deploy-agent.template
```

Comment:
Let's add comments for each of these things explaining why we need it and when to change the default values.

Reply:
Removed this section as this is not user configurable